src/Listeners/SslCertificateExpiresSoonListener.php

```diff
@@ -4,16 +4,23 @@
 
 use Illuminate\Support\Facades\Notification;
 use Taecontrol\MoonGuard\Repositories\UserRepository;
+use Taecontrol\MoonGuard\Notifications\SlackNotifiable;
 use Taecontrol\MoonGuard\Events\SslCertificateExpiresSoonEvent;
 use Taecontrol\MoonGuard\Notifications\SslCertificateExpiresSoonNotification;
 
 class SslCertificateExpiresSoonListener
 {
     public function handle(SslCertificateExpiresSoonEvent $event): void
     {
-        Notification::send(
-            UserRepository::all(),
-            new SslCertificateExpiresSoonNotification($event->sslCertificateCheck)
-        );
+        $channels = config('moonguard.notifications.channels');
+
+        foreach ($channels as $channel) {
+            $users = ($channel === 'slack') ? new SlackNotifiable() : UserRepository::all();
```

Comment:
Rename this variable into:

```suggestion
            $notifiables = ($channel === 'slack') ? new SlackNotifiable() : UserRepository::all();
```